src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Overview/ThorchainSaversOverview.tsx

```diff
@@ -241,6 +242,29 @@ export const ThorchainSaversOverview: React.FC<OverviewProps> = ({
       .length,
   )
 
+  const isOpportunityMature = useMemo(() => {
```

Comment:
stylistic: this only applies to RUNEPool. Can we use a more declarative name here as well e.g

```suggestion
  const isRunepoolWithdrawUnlocked = useMemo(() => {
```

Reply:
Changed in [4abc315](https://github.com/shapeshift/web/pull/7472/commits/4abc3154365f8295c06c2fb383b10d762be64282)